ui/DesignSystem/Component/ConnectionStatusIndicator.svelte

```diff
@@ -18,45 +18,60 @@
 </script>
 
 <style>
-  a {
+  .item {
+    width: var(--sidebar-width);
+    height: 32px;
+    margin-bottom: 16px;
+    position: relative;
     display: flex;
-    align-items: center;
     justify-content: center;
-    width: 100%;
-    height: 100%;
+    align-items: center;
+    cursor: pointer;
+  }
+
+  .indicator:hover:before {
+    position: absolute;
+    content: "";
+    width: 4px;
+    height: 32px;
+    background-color: var(--color-foreground-level-5);
+    top: 0px;
+    left: 0px;
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+  }
+
+  .indicator :global(div:hover svg) {
+    fill: var(--color-secondary);
   }
 </style>
 
 <Remote {store} let:data={status}>
   <div>
     {#if status.type === StatusType.Online}
       <Tooltip value={`You’re connected to ${peerCount(status.connected)}`}>
-        <!-- svelte-ignore a11y-missing-attribute -->
-        <a>
+        <div class="item indicator" data-cy="search">
```

Comment:
```suggestion
        <div class="item indicator" data-cy="connection-status">
```

also for the changes below